webapp/src/components/SingleStudy/ExportFilterModal/SingleSelect.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React from 'react';
+import { Select, MenuItem, InputLabel, FormControl, Input } from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+
+interface PropTypes {
+    style?: CSSProperties;
+    fullWidth?: boolean;
+    list: Array<string>;
+    value: string;
+    label: string;
+    onChange: (value: string) => void;
+}
+
+const MultipleSelect = (props: PropTypes) => {
```

Comment:
Rename component so that it reflects the name of the file (this is a SingleSelect ?)